home/templates/pages/about.html

```diff
@@ -11,10 +11,7 @@ <h1 class="display-2 mb-4">
             Full-Service <br class="d-none d-md-inline" />Cyber Security Agency
           </h1>
           <p class="lead mb-4 text-muted">
-            Hardhat can help you secure your webapps, review your code,

```

Comment:
Whats this about can you return it back to its normal state or add any new version of it aswell